### DISCUSSION TITLE
Vertically align page titles (left) with action buttons (right)?

### DISCUSSION THREAD

**Discussion by mariomarroquim** [avo-hq/avo#351] [Feature requests]
AVO’s design is just beautiful, especially for those coming from ActiveAdmin, but the page title vertical align bothers me a bit. Does someone feel the same? Is it something to improve? Here aree some examples:

![avodemo herokuapp com_avo_resources_posts(Laptop with MDPI screen)](https://user-images.githubusercontent.com/88173/114438132-bd1e9180-9b9d-11eb-9531-2f1388a10e74.png)

![avodemo herokuapp com_avo_resources_posts(Laptop with MDPI screen) (1)](https://user-images.githubusercontent.com/88173/114438138-c14aaf00-9b9d-11eb-94eb-46fd9b928e05.png)

**Comment by adrianthedev** [comment]
Hello, 

Thank you for the kind words.
We are open to suggestions.

We were [thinking](https://github.com/avo-hq/avo/discussions/308) about allowing the developer to customize the subheading at some point.
